### PR TITLE
add description of reddit tasks

### PR DESCRIPTION
**Pull request by jlabeit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Implementation
+Once the setup is complete you can implement your tests in the `./tests` directory. If you are new to playwright checkout [this tutorial](https://playwright.dev/docs/writing-tests) for some basic instruction on how to write tests. 
+
+If you want to give the bonus task a try you can add additional CI workflows in the `.github/workflows` directory. If you are new to github workflows checkout [this resource](https://docs.github.com/en/actions/using-workflows) - it explains the basics well. 
+
+
+# Notes
+* The deadline for the task is 3 days.
+* You can check if all your tests pass by running `yarn test`.
+* Once you finish the task, you should commit your changes to a separate branch and create a pull request. You can then see via github if the tests or other checks pass or fail and fix them accordingly.
```
Comment (david-paulus-saad) [inline README.md:61]: If there are multiple candidates in-parallel, will they be able to see branches of each other?

Comment (jlabeit) [reply README.md:61]: I think how we usually share tasks with candidates is that we create a new repo for each candidate based in this template repo. You can see an example [here](https://github.com/impargo/pet-2021-chantalchawki/pull/1). All the repos are private and just the candidate is invited to it who should work on the task.

Comment (david-paulus-saad) [reply README.md:61]: that makes sense

**Review by david-paulus-saad — APPROVED** [review]

**Comment by david-paulus-saad** [comment]
Looks good to me :100:

**Pull request merged**